backend/yelpAPI/fakeDataAPI.py

```diff
@@ -35,19 +39,62 @@ def get_business():
 
 
 # route gets 50 businessses from yelp API
-@app.route("/random", methods=["GET"])
-def get_our_own_businesses():
-    # Grab a random state from states.py
-    random_state = random.choice(states.us_states)
-    # Use query filter to get 50 businesses from the random state provided, can change limit query to X amount
+@yelp_bp.route("/api/<latitude>/<longitude>", methods=["GET"])
+def get_our_own_businesses(latitude, longitude):
+    # Get a random integer for the offset
+    random_int = str(random.randint(0, 500))
+    # Use query filter to get 50 businesses from provided latitude and longitude, can change limit query to X amount
+    # offset is from integer above, to get more random options
     response = requests.get(
-        URL + "businesses/search?location=" + random_state + "&limit=50",
+        URL

```

Comment:
I think you should have this as a formatted string instead of doing + for every part of the URL.